code/datums/traits/negative.dm

```diff
@@ -139,7 +147,7 @@
 				heirloom_type = pick(/obj/item/clothing/neck/stethoscope, /obj/item/book/manual/wiki/security_space_law)
 			//RnD
 			if("Research Director")
-				heirloom_type = /obj/item/toy/plush/slimeplushie
+				heirloom_type = pick(/obj/item/toy/plush/slimeplushie, /obj/item/phone)
```

Comment:
WHY THE PHONES

Reply:
They're clunky, cannot be worn and are useless. As a family heirloom should be?